### PR TITLE
Barber Scissors: The Assistant of Seville

### PR DESCRIPTION
**Pull request by GreatBigFailure**
Adds barber scissors and puts them in the barbervend. Sprite by Bomber Harris. 
Barber scissors work like pocket mirrors but you can change other people's hairstyles with them, not just your own. There's a 3 second delay on this and creates a warning message so no surprise haircuts. You also can't cut your own hair. Remember, you should always go to the barber with the second best haircut. 
Finally, this replaces the lipstick in barbervend's contraband with the pocket mirror. Mostly I just didn't like the lipstick being there because we have a makeup vendor already.

bigscissor2

barberscreenshot

-->
:cl:
 * rscadd: Barbervends now sell scissors which allow actual barbering to take place.

### CODE REVIEW

**Comment by PavelCIA** [comment]
on my way to give every prisoner a floorlength braid as warden so they cant run away

**Comment by calzilla1** [comment]
Some of the more "gaudy" hair styles should be locked behind getting a haircut. Hey, looking like a rejected anime protagonist ain't cheap.

**Comment by Eneocho** [comment]
> 
> 
> Some of the more "gaudy" hair styles should be locked behind getting a haircut. Hey, looking like a rejected anime protagonist ain't cheap.

The haircut gang nerf... Is this the end of the meterlong pompadour hairdo?

**Comment by Eneocho** [comment]
Sweeney Todd gimmicks soon?

**Comment by BomberBro** [comment]
The scissors sprite was a WIP from awhile ago, if anyone has a better sprite feel free to throw it into the gauntlet.

File: code/game/objects/items/weapons/cosmetics.dm [hunk 1]
```diff
@@ -567,6 +568,25 @@
...
+	icon_state = "barber_scissors"
+	sharpness = 1
+	sharpness_flags = SHARP_TIP | SHARP_BLADE
+
+/obj/item/weapon/pocket_mirror/scissors/shatter()
+	return
+
+/obj/item/weapon/pocket_mirror/scissors/attack(atom/target, mob/user)
+	if(ishuman(target) && target != user)
+		user.visible_message("<span class='warning'>[user] begins changing [target]'s style!</span>")	//Chitin, slime, etc. Can't use the word "hair"
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/cosmetics.dm]: `danger` is better for this  purpose, as it is bold

**Review by Kurfursten — CHANGES_REQUESTED** [review]
otherwise looks good to go

**Review by Kurfursten — APPROVED** [review]

**Pull request merged**